### PR TITLE
Create a Readonly Propagator to match the golang "One Way" propagator.

### PR DESCRIPTION
**Pull request by muncus**
Also adds usage information to README.rst for both propagators.

### CODE REVIEW

**Comment by muncus** [comment]
can I get a /gcbrun? 
also, would you prefer this be named "one way" propagator to keep consistency? I thought Readonly was more descriptive but i also value consistency 😄

**Comment by muncus** [comment]
I've renamed it to OneWay for consistency with go and java propagators, and fixed the type issue.
I believe this is ready for review (after /gcbrun).

**Comment by muncus** [comment]
ping.

File: opentelemetry-propagator-gcp/README.rst [hunk 1]
```diff
@@ -18,8 +18,51 @@ Installation
...
+    from opentelemetry.propagate import set_global_textmap
+    from opentelemetry.propagators.composite import CompositePropagator
+    from opentelemetry.propagators.cloud_trace_propagator import (
+        CloudTraceOneWayPropagator,
+    )
+    set_global_textmap(CompositePropagator([
+        CloudTraceOneWayPropagator(),
+        propagate.get_global_textmap()
+    ]
+    ))
```
Comment (aabmass) [inline opentelemetry-propagator-gcp/README.rst:40]: formatting nit
```suggestion
    set_global_textmap(
        CompositePropagator([
            CloudTraceOneWayPropagator(),
            propagate.get_global_textmap(),
        ]),
    )
```

**Review by aabmass — APPROVED** [review]

**Pull request merged**